sdks/python/build.gradle

```diff
@@ -226,6 +228,26 @@ task directRunnerIT(dependsOn: 'installGcpTest') {
   }
 }
 
+task precommitIT(dependsOn: ['installGcpTest', 'sdist']) {
+  doLast {
+    // List of integration tests running in Python PreCommit.
+    def precommitTests = [
+        "apache_beam.examples.wordcount_it_test:WordCountIT.test_wordcount_it",
+        "apache_beam.examples.streaming_wordcount_it_test:StreamingWordCountIT.test_streaming_wordcount_it",
+    ]
+    def testOpts = [
+        "--tests=${precommitTests.join(',')}",
+        "--processes=4",
+        "--process-timeout=1800",   // Total timeout includes all tests run.
+    ]
+
+    exec {
+      executable 'sh'
+      args '-c', ". ${envdir}/bin/activate && ./scripts/run_integration_test.sh --test_opts \"${testOpts.join(' ')}\""
```

Comment:
After parallelizing this task, please add a copy of it in `sdks/python/precommit/directrunner/build.gradle` with the option `--runner=TestDirectRunner`.

Reply:
sg. I'd love to put it in next PR.